### PR TITLE
Utility documentation

### PR DESCRIPTION
**Pull request by rinkk**
Added documentation for

* createIntermediateRasters
* GoCadTSurfaceReader
* Mesh2Raster
* Mesh2Shape
* TecPlotTols

### CODE REVIEW

File: Applications/Utils/FileConverter/TecPlotTools.cpp [hunk 1]
```diff
@@ -418,14 +418,18 @@ int main(int argc, char* argv[])
             "Copyright (c) 2012-2019, OpenGeoSys Community "
             "(http://www.opengeosys.org)",
         ' ', GitInfoLib::GitInfo::ogs_version);
-    TCLAP::ValueArg<std::string> input_arg("i", "input-file", "TecPlot input file", true, "", "string");
-    cmd.add(input_arg);
-    TCLAP::ValueArg<std::string> output_arg("o", "output-file", "output mesh file", false, "", "string");
-    cmd.add(output_arg);
-    TCLAP::SwitchArg split_arg("s", "split", "split time steps into seperate files");
+    TCLAP::SwitchArg split_arg("s", "split",
+        "split time steps into seperate files");
```
Comment (TomFischer) [inline Applications/Utils/FileConverter/TecPlotTools.cpp]:
```suggestion
        "split time steps into separate files");
```

File: web/content/docs/tools/fileio/Mesh2Shape/index.pandoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   -o <output_file.shp>,  --output-file <output_file.shp>
+     (required)  Esri Shapefile (*.shp)
+```
+
+## Simple example
+
+**Input data:**
+
+![](./Mesh2Shape-input.png){ width=66% }
+2D surface mesh mit scalar data assigned to cells, here displayed via the OGS Data Explorer. In this particular case, the simulation result of groundwater flow simulation (originally assigned to mesh nodes) has been converted onto cells via VTK's PointToCell-Filter.
```
Comment (TomFischer) [inline web/content/docs/tools/fileio/Mesh2Shape/index.pandoc]:
```suggestion
2D surface mesh with scalar data assigned to cells, here displayed via the OGS Data Explorer. In this particular case, the simulation result of groundwater flow simulation (originally assigned to mesh nodes) has been converted onto cells via VTK's PointToCell-Filter.
```

File: web/content/docs/tools/fileio/Mesh2Shape/index.pandoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## Introduction
+
+Converts a 2D surface mesh into a shapfile such that each element is represented by a polygon. Cell attributes are transferred onto shape polygons while point attributes are ignored.
+
+The tool is part of the official [OpenGeoSys git repository](https://github.com/ufz/ogs).
+
+## Usage
+
+```bash
+   Mesh2Shape.exe  -i <input_file.vtu> -o <output_file.shp>
```
Comment (TomFischer) [inline web/content/docs/tools/fileio/Mesh2Shape/index.pandoc]: In order to stay consistent with the other tools:
```suggestion
   Mesh2Shape  -i <input_file.vtu> -o <output_file.shp>
```

File: web/content/docs/tools/fileio/GocadTSurfaceReader/index.pandoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Another data type, SGRID (Structured Grid, usually saved to *.sg files) can be converted via the [GoCadSGridReader](../../meshing/gocadsgridreader).
+
+Parsers for additional GoCAD-datasets may be added in the future.
+
+The tool is part of the official [OpenGeoSys git repository](https://github.com/ufz/ogs).
+
+## Usage
+
+```bash
+   GocadTSurfaceReader.exe  -i <filename.ts> -o <output dir> [-l] [-s] [-b]
```
Comment (TomFischer) [inline web/content/docs/tools/fileio/GocadTSurfaceReader/index.pandoc]: In order to stay consistent with other tools:
```suggestion
   GocadTSurfaceReader  -i <filename.ts> -o <output dir> [-l] [-s] [-b]
```

File: web/content/docs/tools/meshing/vtu2grid/index.pandoc [hunk 5]
```diff
@@ -17,8 +17,8 @@ The tool is part of the official [OpenGeoSys git repository](https://github.com/
 ## Usage
 
 ```bash
-   Vtu2Grid.exe  -i <input.vtu> -o <output.vtu> -x <floating point number>
-                 [-y <floating point number>] [-z <floating point number>]
+   Vtu2Grid  -i <input.vtu> -o <output.vtu> -x <floating point number>
```
Comment (endJunction) [inline web/content/docs/tools/meshing/vtu2grid/index.pandoc]: Same mismatch in other new documentation snippets; Sometimes `.exe` sometimes `linux`.

**Pull request merged**